### DISCUSSION TITLE
Low-cost automated powder dispensers

### DISCUSSION THREAD

**Discussion by sgbaird** [sparks-baird/self-driving-lab-demo#92] [Demo extensions and design]
Worth reading the articles here to start out:
- https://hackaday.com/tag/bulk-material/

Another keyword: electronic powder trickler

## Open-source/DIY
- RADIN https://www.thingiverse.com/thing:3490082 ([mentioned](https://twitter.com/ChristophVoelk2/status/1578476708478320640) by @BAMcvoelker)
- Open Powder https://www.thingiverse.com/thing:3870337 (designed for ammunition refill, [mentioned](https://twitter.com/ChristophVoelk2/status/1578719224951844865) by @BAMcvoelker)

## Congruent Commercial Products
- [WiFi-enabled Powder dispenser for baby formula](https://www.amazon.com/Baby-Brezza-Formula-Advanced-Dispenser/dp/B09K4KVN3Z) ($300)
- Bird feeders
  - https://www.amazon.com/Wingscapes-AutoFeeder-Automatic-Bird-Feeder/dp/B0083H9HBU
- Fish feeders
  - https://www.amazon.com/lychee-Upgrade-Automatic-Aquarium-Dispenser/dp/B07MQ8KDCL (~$30)
- misc
  - https://www.amazon.com/powder-dispenser/s?k=powder+dispenser

@BAMcvoelker mentioned the issue with many of the feeders (and also the baby formula dispenser) having preset settings - not being able to specify an arbitrary weight setting.

##
[Twitter thread](https://twitter.com/ChristophVoelk2/status/1578476708478320640)

Related:
- #77 
- #153

**Comment by iterateccvoelker** [comment]
Great summary. Thanks for sharing 🫶

**Reply by sgbaird** [reply]
@BAMcvoelker curious to know if you have any updates on this front?

**Reply by iterateccvoelker** [reply]
Hi Sterling, thanks for your continued interest. I have printed everything out and ordered all the parts, but haven't gotten as far as actually installing all the parts. The main reasons are that we haven't been able to find a student assistant yet (a new person is starting soon). Second, the geometry is not compatible with our robot. The sample tubes don't fit quite as well under the front nose. 

For an MVP, we will still build this or a similar system, but measure the powders by hand and just fill a sample hotel with different weight fractions. This is not the best solution, but we are just a prototype in science.

**Reply by sgbaird** [reply]
Thanks for the update!

**Comment by sgbaird** [comment]
[Comment by Josh Schrier](https://twitter.com/JoshuaSchrier/status/1578965917936930816):

> Maciej Haranczyk and I went down an internet rabbit hole of automated powder feeders (which these open-source designs are modeled on).  Commercial ones are relatively cheap (~$200-$400) for customers that care about precision. https://midwayusa.com/powder-measures-scales-and-tools/br?cid=701

e.g. [RCBS ChargeMaster Link Digital Powder Scale and Dispenser](https://www.midwayusa.com/product/1024207880?pid=657660) ($300, 6.5 mg precision, 22.7 grams max, 129.5 gram capacity)
